### ISSUE TITLE
Update alevin-fry images to 0.4.1 (and maybe salmon?)

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/alsf-scpca#118]
In response to some comments from us https://github.com/COMBINE-lab/alevin-fry/issues/22#issuecomment-885195051 `alevin-fry` now has better `.json` output files, so we should update. When we do that, some of the output files will change, but for the better! However, this will temporarily break scpcaTools (filing that issue separately).

We may want to wait for a coming version bump for salmon that will come in parallel, which adds back the `cmd_info.json` file for `--rad`  outputs.

One other note along with that is that we can remove the t2g file (`--tgMap`) from the `salmon alevin` call, as this is no longer needed with `--rad`
